src/components/TableOfContents.tsx

```diff
@@ -11,7 +11,7 @@ import styles from "./TableOfContents.module.css";
 function TableOfContents({ headers, data }) {
   const handleRowClick = (firstColumnValue) => {
     // Convert first column value to a valid ID format
-    const sectionId = firstColumnValue.toLowerCase().replace(/\s+/g, "");
+    const sectionId = firstColumnValue.toLowerCase().replace(/\s+/g, "-");
```

Comment:
so that table of contents section names can have spaces and still be linked to a specific section on the page